bgpd/bgp_io.c

```diff
@@ -447,7 +448,7 @@ static uint16_t bgp_read(struct peer *peer)
 	size_t readsize; // how many bytes we want to read
 	ssize_t nbytes;  // how many bytes we actually read
 	uint16_t status = 0;
-	static uint8_t ibw[BGP_MAX_PACKET_SIZE * BGP_READ_PACKET_MAX];
+	uint8_t ibw[BGP_PACKET_SIZE(peer->bgp) * BGP_READ_PACKET_MAX];
```

Comment:
I assume you checked it's safe to stack allocate whatever size this could be

Reply:
Do you mean going above 8mb (default stack size)? This would end up no more than 64k*10.

Reply:
subsequent code doesn't rely on the 0 initialization either, right? pretty sure not